src/civetweb.c

```diff
@@ -8107,6 +8112,29 @@ mg_websocket_write(struct mg_connection *conn,
 		headerLen = 10;
 	}
 
+	if(mask) {
+		/* add mask */
+		srand(time(NULL));
```

Comment:
As to my knowledge, it is system dependent if this function affects the entire process or just the calling thread (see [here](http://stackoverflow.com/questions/10159728/does-one-need-to-call-srand-c-function-per-thread-or-per-process-to-seed-the-r)). 
For several reasons I would like to avoid using srand and rand - but this is something I can change afterwards and replace it by another pseudo-random number generator.